src/utils/formValidation.test.ts

```diff
@@ -15,6 +15,16 @@ describe('formValidation', () => {
     });
   });
 
+  describe('mustBeEmail', () => {
+    test('should return undefined if the email is valid', () => {
+      expect(mustBeEmail('test@example.com')).toBeUndefined();
+    });
+
+    test('should return an error message if the email is invalid', () => {
+      expect(mustBeEmail('test@.com')).toMatchSnapshot();
```

Comment:
I think we should start relying less on snapshots if possible:
- It's tempting to tell jest to just update all snapshots when something fails -> we might miss problems
- You cannot see the actual "contract" or expected values from the unit test -> harder to understand the function/module under test and the unit test doesn't server as documentation of the contract or API anymore

Reply:
👍 Not strictly in the scope of this PR, but changed them anyway

Reply:
Good 👍